src/jrd/SysFunction.cpp

```diff
@@ -4583,6 +4589,25 @@ dsc* evlGetContext(thread_db* tdbb, const SysFunction*, const NestValueArray& ar
 			GuidToString(guidBuffer, &dbb->dbb_guid);
 			resultStr = string(guidBuffer);
 		}
+
+        else if (nameStr == PAGES_ALLOCATED)
+        {
+
+            resultStr.printf("Number of DB pages allocated = %" SQUADFORMAT, PageSpace::actAlloc(dbb));
+        }
+
+        else if (nameStr == PAGES_USED)
+        {
+
+            resultStr.printf("Number of DB pages used = %" SQUADFORMAT, PageSpace::usedPages(dbb));
+        }
+
+        else if (nameStr == PAGES_FREE)
+        {
+
+            resultStr.printf("Number of DB pages free = %" SQUADFORMAT, PageSpace::maxAlloc(dbb) - PageSpace::usedPages(dbb));
+        }
+
```

Comment:
"Number of ..." text is not needed here, only the number should be returned.

Reply:
And given that `PageSpace::*` functions return `ULONG`, why `SQUADFORMAT`?

Reply:
While formally the new code is correct, I'd suggest to use ULONGFORMAT instead of "lu" like it's done for SESSION_IDLE_TIMEOUT or STATEMENT_TIMEOUT.